src/components/content/Content.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import CreateTask from "src/components/tasks/CreateTask";
+import TaskItems from "src/components/tasks/TaskList";
+import { addTasks } from "src/store/actions/taskActions";
+import {
+  ALT_ADD_TEXT,
+  CLASS_BUTTON_CREATE,
+  ICON_ADD,
+  TEXT_CREATE,
+} from "src/utils/constants/constants";
+import { Button } from "src/components/button/Button";
```

Comment:
The reason I asked https://github.com/rsa-wtag/Todos/pull/1/files#r1200497097 is because you are importing CreateTask, TaskItems and Button differently. 

We are not going to import other things from Button component. So I would make it default export.